FusionMCPBridge/handlers/design/geometry_impl2.py

```
@@ -0,0 +1,642 @@
+                "Einheit": str(param.unit),
+                "Expression": str(param.expression) if param.expression else ""
+            })
+    return model_params
+
+
+def set_parameter(design, ui, name, value):
+    """Set a parameter value"""
+    try:
+        param = design.allParameters.itemByName(name)
+        param.expression = value
+    except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)